docs/pages/versions/unversioned/sdk/camera-legacy.mdx

```diff
@@ -9,20 +9,19 @@ platforms: ['android*', 'ios*', 'web']
 
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
-import PrereleaseNotice from '~/components/PrereleaseNotice';
+import { SnackInline } from '~/ui/components/Snippet';
 import {
   ConfigReactNative,
   ConfigPluginExample,
   ConfigPluginProperties,
 } from '~/components/plugins/ConfigSection';
 import { AndroidPermissions, IOSPermissions } from '~/components/plugins/permissions';
 
-<PrereleaseNotice>
-  This page documents an upcoming version of the Camera library. **To try it, switch to
-  `expo-camera/next` imports in your code.**
-</PrereleaseNotice>
+> **warning** This is the legacy version of the `expo-camera` package which will no longer be receiving updates. If you are starting a new project, we recommend using the latest version. You can find it [here](/versions/unversioned/sdk/camera).
```

Comment:
```suggestion
> **warning** This is the legacy version of the `expo-camera` package which will no longer be receiving updates. If you are starting a new project, we recommend using [the latest version of expo-camera](/versions/unversioned/sdk/camera).
```

unrelated to the above change - we probably don't want to send people to unversioned docs here but we also don't have a good solution for this intermediate state. perhaps we could link to v51.0.0 explicitly here? it'd be a broken link for now until we version the docs, but that's probably fine - it'll work once users access it. cc @amandeepmittal if you have any preference

Reply:
Updated 👍